bin/dswx_hls.py

```diff
@@ -51,6 +51,7 @@ def main():
         args.output_file,
         hls_thresholds = hls_thresholds,
         dem_file=args.dem_file,
+        dem_description=args.dem_description,
```

Comment:
Yes, good point. We need to decide what needs to be passed in this argument.

Reply:
I'm adding a comment to the DSWx-HLS product specs, and I'll resolve this comment here because it's not part of this PR, but thank you for the comment (we definitely need to define how the PGE will populate these fields).